netdisco/discoverables/hikvision.py

```diff
@@ -0,0 +1,13 @@
+"""Discover SABnzbd servers."""
```

Comment:
Stale comment.

This PR also contains unrelated commits.

Reply:
Good spot I will fix now.